### DISCUSSION TITLE
Lighthouse HTC vs Valve basestations

### DISCUSSION THREAD

**Discussion by TommiAux** [bitcraze/discussions#1367] [Q&A]
Hi,

a few days ago I was trying a setup with both HTC and Valve lighthouse basestations. At first, I thought there were no differences between those two. However, upon closer inspection I could make out a few small differences.

Variant 1:
- White status LED on normal operation
- Power, USB port and 3.5 mm audio port (?) on the back

Variant 2:
- Green status LED on normal operation
- Only power and USB port on the back

Both variants are v2 basestations. I'm not entirely sure, but I think variant 1 is an HTC basestation and variant 2 is a valve basestation. Variant 1 was purchased through the Bitcraze store and was part of the Lighthouse swarm bundle. These also work perfectly fine.

The problem I am facing now, is that the basestations of variant 2 are not recognized by my lighthouse decks. I was able to set the channel of the basestations, but the recieve-indicators in the lighthouse tab of the cfclient stay red. In addition, they interfere with the other basestations.

Perhaps someone has made similar observations and knows more about this?

Many thanks and best regards
Tommi

**Comment by ataffanel** [comment]
Thank you for sharing the problem, I would love to reproduce it. Where and when did you buy the non-working basestation?

I am assuming there is someting with firmware version. The version can be identified by connecting the basestation via USB, opening the usb serial port of it and typing ```id```. For me, on a working base-station, I get: 
```
lhtx> id
Device Name: DEFAULT
Serial Number: 312361B0
Firmware Build: 0.1.8138615 2018-01-18 04:07:10 <buildsystem@fw-keil5-win32>
Radio Build: FW 1.1.6277248     HW 0.0, 3265mV  <buildsystem>
FPGA Code Version: 6
OOTX Model: 14
```

Is it possible for you to do the same and send the result?

**Comment by TommiAux** [comment]
Hi @ataffanel

Thank you for your reply. Today I set up the same setup again and miraculously it worked this time. I can't tell what I did differently this time. The base stations have channels 5, 6 and 7 and cover a flight area of approx. 5 x 4 metres. I have built the firmware for up to 8 base stations.

Nevertheless, I read out the versions for two of the different base stations as described by you.

Variant 1:
```
id
Device Name: DEFAULT
Serial Number: 3BA26B45
Firmware Build: 0.1.8138615 2018-01-18 04:07:10 <buildsystem@fw-keil5-win32>
Radio Build: FW 1.1.6277248     HW 0.0, 3284mV  <buildsystem>
FPGA Code Version: 6
OOTX Model: 14
lhtx>
```
Variant 2:
```
id
Device Name: DEFAULT
Serial Number: 8FF6FF80
Firmware Build: 1.8.2004742 2019-07-24 16:06:45 <buildsystem@fw-keil5-win32>
Radio Build: FW 2.9.2004771 HW 0.0, 3265mV  BTLDR 3.4.3782793        <buildsystem>
FPGA Code Version: 7
OOTX Model: 15
lhtx>
```

So everything is working fine. I hope I did not waste too much of your time :)

Thanks again and best regards
Tommi

**Reply by ataffanel** [reply]
Thanks for the update and glad that it worked in the end!

It is interesting that the new basestation have a different firmware, I am going to try to either update some of our basestations or buy a newer one, I do not expect the protocol to change but it would be good for us to test as much as possible with different basestation versions.
